dolphinscheduler-master/src/main/resources/logback-spring.xml

```diff
@@ -74,4 +74,4 @@
         <appender-ref ref="TASKLOGFILE"/>
         <appender-ref ref="MASTERLOGFILE"/>
     </root>
-</configuration>
+</configuration>
```

Comment:
Same here.

Reply:
> Same here.
I got it

Reply:
How should this be repaired? It seems that the file has not changed in the end.